src/components/screens/OperatingCompanyInfo.js

```diff
@@ -104,22 +102,21 @@ export const OperatingCompanyInfo = ({ address, contact, name, webUrls, openWebS
               <Icon xml={mail(colors.primary)} style={styles.margin} />
               <TouchableOpacity
                 onPress={() => openLink(`mailto:${contact.email}`)}
-                accessibilityLabel="E-mail-Adresse"
-                accessibilityHint="Navigieren zur E-mail-App"
+                accessibilityLabel={`(E-mail-Adresse) (${contact.email}) (Taste) (Wechselt zur Email-App)`}
               >
                 <RegularText primary>{contact.email}</RegularText>
               </TouchableOpacity>
             </InfoBox>
           )}
           {!!contact.fax && (
-            <InfoBox accessible={true}>
+            <InfoBox>
               <RNEIcon
                 name="print"
                 type="material"
                 color={colors.primary}
                 iconStyle={{ marginRight: normalize(10) }}
               />
-              <RegularText primary accessibilityLabel="Fax-nummer">
+              <RegularText primary accessible={true} accessibilityLabel={`(Fax) (${contact.fax}`}>
```

Comment:
nice that you have put `Fax` inside. otherwise it would be just a number without context